docs/tutorials/adding-a-support-desk-button.md

```diff
@@ -0,0 +1,243 @@
+    apos.bus.$on('admin-menu-click', (item) => {
+      // Check if our support desk button was clicked
+      if (item === 'support-desk') {
+        openSupportWidget();
+      }
+    });
+  });
+
+  function openSupportWidget() {
+    // Call the appropriate API based on which support desk you're using
+    // The exact method will vary by provider - check their documentation
+
+    // For most widget-based help desks
+    if (window.supportWidget) {
+      window.supportWidget.open();
+    }
+
+    // Add an announcement for screen readers (accessibility improvement)
+    const announcement = document.createElement('div');
+    announcement.setAttribute('aria-live', 'polite');
+    announcement.textContent = 'Support desk is now open';
+    document.body.appendChild(announcement);
+
+    // Remove after announcement is read
+    setTimeout(() => {
+      document.body.removeChild(announcement);
+    }, 3000);
+  }
+};
+```
+  <template v-slot:caption>
+    modules/asset/ui/src/index.js
+  </template>
+</AposCodeBlock>
+
+This approach ensures that when a content manager clicks the support button in the admin bar, the third-party support desk widget appears, allowing them to get help without leaving the CMS interface.
+
+> **Note**: You'll need to replace `window.supportWidget.open()` with the actual method provided by your support desk solution. Check their developer documentation for the correct method name and parameters.
+
+### Including the Support Desk Script
+
+You'll need to add the initialization code for your chosen support desk solution. This can be done in your layout template or in your UI JavaScript file, depending on your preference and the support tool's requirements.
+
+#### General Pattern
+
+Regardless of which support desk solution you use, the implementation typically follows this pattern:
+
+1. Only load the support desk for authenticated users
+2. Include the vendor's JavaScript
+3. Configure the widget to hide the default launcher
+4. Pass user information when available
+
+#### Template-Based Implementation
+
+This approach works well for simpler implementations:
+
+<AposCodeBlock>
+
+```nunjucks
+{% block extraBody %}
+  {{ super() }}
+
+  {% if data.user %}
+    {# Load the support desk script only for authenticated users #}
+    <script src="https://cdn.example.com/support-desk.js?key=YOUR_API_KEY" async></script>
+    <script>
+      // Configure your support desk
+      // This example is generic - check your provider's documentation for specifics
+      window.supportDeskConfig = {
+        // Common configurations:
+        hideLauncher: true,
+        user: {
+          name: '{{ data.user.title }}',
+          email: '{{ data.user.email }}'
+        }
+      };
+    </script>
+  {% endif %}
+{% endblock %}
+```
+  <template v-slot:caption>
+    views/layout.html
+  </template>
+</AposCodeBlock>
+
+#### JavaScript-Based Implementation
+
+For more control, you can load the script programmatically:
```

Comment:
Make this logic part of the ui/apos/apps/SupportDesk.js file.

Reply:
Updated.